Cargo.toml

```diff
@@ -24,6 +24,8 @@ targets = ["aarch64-apple-darwin", "x86_64-apple-darwin", "x86_64-unknown-linux-
 pr-run-mode = "plan"
 # Whether to install an updater program
 install-updater = false
+# Suppress dist failure because we bumped ubuntu-20.04 to ubuntu-22.04
+allow-dirty = ["ci"]
```

Comment:
Can this be removed again after the next successful release?

Reply:
I don't know yet. `cargo-dist` is now unmaintained because the company behind it failed. They have been talking about handing it over to the community but it hasn't happened yet. Ideally they just do that and `cargo-dist` gets fixed and then I don't have to apply the workarounds myself anymore.